.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,21 +10,24 @@
 
 
 ## How Has This Been Tested?
-<!--- Please describe in detail how you tested your changes. -->
-<!--- Include details of your testing environment, and the tests you ran to -->
-<!--- see how your change affects other areas of the code, etc. -->
+ - [ ] I have tested the markdown appearance with Docker (Refer to the [local development instructions](https://github.com/openfaas/docs#local-development))
+<!--- Please include screenshots of the main areas of change.-->
+
 
 ## Types of changes
-<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+<!--- What types of changes are included? Put an `x` in one the boxes: -->
+<!--- Where new content covers a new feature please reference the PR for this feature -->
+
+- [ ] New Content
+- [ ] Content Correction
+- [ ] Content Improvement
+- [ ] Non-content (config/templates/mkdocs/CSS)
 
 ## Checklist:
 <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
-<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 - [ ] My code follows the code style of this project.
-- [ ] My change requires a change to the documentation.
-- [ ] I have updated the documentation accordingly.
+- [ ] I have tested any new instructions
+- [ ] I am the author 
```

Comment:
I think this declaration makes sense, but some people (international speakers, perhaps) may assume this means that they created the PR. Can we be more specific, but remain concise?

Do we need both?

What about:

`- [ ] I have included  copyrighted materials from third parties.`
`- [ ] I have permission to use copyrighted materials from third parties.`

Reply:
To some extent this is also covered under the DCO

Reply:
addressed